.github/workflows/run-host-tests.yml

```diff
@@ -51,14 +57,24 @@ jobs:
             # The sdkconfig.ci.linux specifies Linux as the build target with apropriate settings.
             cp sdkconfig.ci.linux sdkconfig.defaults
             idf.py build
-            ./build/${{inputs.app_name}}.elf -r junit -o junit.xml
+            if [ "${{ inputs.run_executable}}" == "false" ]; then
+                echo "Executeable wasn't run"
+                exit 0
+            fi
+
+            if [ "${{ inputs.publish_unit_test_result }}" == "true" ]; then
+              ./build/${{inputs.app_name}}.elf --reporter JUnit::out=junit.xml --reporter console::out=-::colour-mode=ansi
```

Comment:
does it still make sense to add these arguments if we hardcode them in the `app_main()`?

how is it with other host tests? Do they use the same action?